src/test/java/org/opentripplanner/routing/algorithm/filterchain/ItineraryListFilterChainTest.java

```diff
@@ -190,8 +191,12 @@ void transitAlertsTest() {
     chain.filter(List.of(i1, i2, i3));
 
     // Then transitAlertService should have been called with stop and route ids
-    Mockito.verify(transitAlertService, Mockito.atLeastOnce()).getStopAlerts(A.stop.getId());
-    Mockito.verify(transitAlertService, Mockito.atLeastOnce()).getStopAlerts(E.stop.getId());
+    Mockito
```

Comment:
Just out of curiosity, do we need to use mockito here?

Reply:
We check that the `transitAlertService` gets queried with the correct arguments, without needing to have one.